android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -44,7 +44,7 @@ public class RNPurchasesModule extends ReactContextBaseJavaModule implements Upd
     private static final String CUSTOMER_INFO_UPDATED = "Purchases-CustomerInfoUpdated";
     private static final String LOG_HANDLER_EVENT = "Purchases-LogHandlerEvent";
     public static final String PLATFORM_NAME = "react-native";
-    public static final String PLUGIN_VERSION = "4.6.0";
+    public static final String PLUGIN_VERSION = "5.14.0-SNAPSHOT";
```

Comment:
This had the wrong version number so I updated it

Reply:
https://linear.app/revenuecat/issue/SDK-3098/update-to-look-for-semver-versions-instead-of-looking-for-specific